### ISSUE TITLE
[MD028] Intended fix is ambiguous, consider removing fix support

### ISSUE THREAD

**Issue by yi-Xu-0100** [DavidAnson/markdownlint#315]
The problem pointed out the blank line inside block quotes. I used the quick fix provided and found that the fix was to delete the blank line. Whether blank lines can be reserved, that is, use an empty block reference flag, as demonstrated below.

Content before repair: 

``` md
> Some references

> Some references
```

Content after repair:

``` md
> Some references
>
> Some references
```

Or give a configuration to set this repair method.

**Comment by DavidAnson** [comment]
Fixes are meant to be simple, unambiguous, and able to be run without the user's involvement. In this case, you make a good point that the right change to the Markdown is not clear and may take either form depending on what the user wishes. Therefore, I'm thinking it may be best to remove the fix behavior for this rule.￼

**Comment by yi-Xu-0100** [comment]
Thanks. I will fix the problem manually next time.
